Blockzilla/BrowserViewController.swift

```diff
@@ -993,14 +993,45 @@ extension BrowserViewController: URLBarDelegate {
             urlBar.pasteAndGo()
         }
         let urlContextMenu = PhotonActionSheet(actions: [[customURLItem], [pasteAndGoItem, pasteItem, copyItem]], style: .overCurrentContext)
-        urlContextMenu.modalPresentationStyle = .overCurrentContext
-        urlContextMenu.delegate = self
-        darkView.isHidden = false
-        present(urlContextMenu, animated: true, completion: nil)
+        presentPhotonActionSheet(urlContextMenu)
+    }
+    
+    func urlBarDidPressPageActions(_ urlBar: URLBar) {
+        guard let url = urlBar.url else { return }
+        let utils = OpenUtils(url: url, webViewController: webViewController)
+        let items = PageActionSheetItems(url: url)
+        let titleItem = PhotonActionSheetItem(title: UIConstants.strings.pageActionsTitle)
+        let sharePageItem = PhotonActionSheetItem(title: UIConstants.strings.sharePage, iconString: "icon_openwith_active") { action in
```

Comment:
Two main thoughts:
1. Building this outside BVC would require us to keep a reference to the action sheet's super viewcontroller in order to present the action sheet. It would also require us to make the URL bar public in order to be able to copy the text from it.
2. OpenUtils is sort of an inaccurate name. Everything in OpenUtils is now only used for supporting the page action menu.

With this being said, I think that the best options are either keeping it as is or moving the two variables for can open into BVC itself (in fact, these two variables are not used anywhere outside of this function, so I'm not sure they even need to be variables). I'm probably more in favor of the second option, but I might be missing something here.